### DISCUSSION TITLE
启动报错 Cannot find module 'umi/dist/cli/node' 是什么原因？

### DISCUSSION THREAD

**Discussion by x011223** [umijs/dumi#1914] [Q&A] [answered]
``` json
  "@types/node": "^20.0.0",
  "dumi": "^2.2.10",
  "eslint": "^8.48.0",
  "prettier": "^3.0.3"
 ```
 
 pnpm@8.7.5
 
 
![image](https://github.com/umijs/dumi/assets/31724313/02b1747e-8323-458c-8fa9-18b183a854c6)

**Comment by PeachScript** [comment, marked as answer]
已修复，参考：https://github.com/umijs/dumi/issues/1916#issuecomment-1736597602
